config/CUSTOM-RULES.md

```diff
@@ -129,7 +143,33 @@ Custom rules can have more complicated behavior. They can also include other rul
 
 `OHKO Clause` - ban one-hit KO moves (Fissure, Sheer Cold, etc)
 
-And more (TODO: finish this list)
+`Evasion Moves Clause` - ban moves that directly boost evasion (Double Team, Minimize)
+
+`Evasion Abilities Clause` - ban abilities that boost evasion (Sand Veil, Snow Cloak)
+
+`Moody Clause` - ban the ability Moody
+
+`Swagger Clause` - ban the move Swagger
+
+`NFE Clause` - ban Pokemon that are fully evolved or can't evolve
+
+`CFZ Clause` - ban the use of crystal-free Z-Moves
```

Comment:
```suggestion
`CFZ Clause` - ban the use of crystal-free Z-Moves (having moves like Devastating Drake hacked directly on the moveset in formats like Hackmons, instead of using Dragon Claw + Dragonium Z)
```